smother/control.py

```diff
@@ -119,9 +131,25 @@ def load(cls, file_or_path):
         result.data = data
         return result
 
+    def _normalize_path(self, path):
+        aliases = None
+        if self.coverage and self.coverage.config.paths:
+            aliases = PathAliases()
```

Comment:
Or better yet, we refactor this into a `_normalize` helper function with no inputs, that normalizes the entire in-memory smother report. Then we only invoke this function when we call `smother combine`, and not during first collection.